packages/server/Dockerfile

```diff
@@ -38,7 +38,7 @@ RUN apt update && apt upgrade -y \
 
 COPY package.json .
 COPY dist/yarn.lock .
-RUN yarn install --production=true \
+RUN yarn install --production=true --network-timeout 1000000 \
```

Comment:
What's the backstory here? 😄

Reply:
We are using the default gh runners and we have plenty of packages to download. Sometimes we hit the default yarn timeout, and it needs to retry. Increasing the timeout will ensure we don't have this unnecessary retries